src/remote_gym/remote_environment_management.py

```diff
@@ -11,26 +18,40 @@
     dm_env_rpc_pb2_grpc,
     spec_manager,
     tensor_spec_utils,
+    tensor_utils,
 )
 from google.rpc import code_pb2, status_pb2
 
+from remote_gym.remote_environment import RemoteArgs
+from remote_gym.repo_manager import RepoManager
 
-def start_as_remote_environment(
-    local_environment: Union[gym.Env, gymnasium.Env],
+
+def create_remote_environment_server(
+    default_args: RemoteArgs,
     url: Text,
     port: int,
     server_credentials_paths: Optional[Tuple[Text, Text, Optional[Text]]] = None,
     enable_rendering: bool = False,
 ) -> grpc.Server:
     """
     Method with which every environment can be transformed to a remote one.
-    Starts the Catch gRPC server and passes the locally instantiated environment.
+    Starts the Catch gRPC server and instantiates environments on new connections.
     Requires credentials to open a secure server port in gRPC. Needs to match the client authentication.
 
-    NOTE: Use the RemoteEnvironment class to connect to a remotely started environment and provide a gym.Env interface.
+    The entrypoint file must define a function `create_environment`:
```

Comment:
I think this documentation is better placed in `RemoteArgs`.

Reply:
What part? The constructor nor the args in this doc is related to RemoteArgs

Reply:
The details about the entrypoint (that it needs to include the `create_environment`).